src/framework/mpas_io.F

```diff
@@ -5919,20 +6074,20 @@ subroutine MPAS_io_put_att_real1d(handle, attName, attValue, fieldname, syncVal,
          allocate(singleVal(size(attValueLocal)))
          singleVal(:) = real(attValueLocal(:),R4KIND)
 #ifdef MPAS_PIO_SUPPORT
-         pio_ierr = PIO_put_att(handle % pio_file, varid, attName, singleVal) 
+        pio_ierr = put_att_pio(handle, varid, attName, singleVal, ierr=ierr)
```

Comment:
I think there may be another space of indentation that's needed here to align with other code.